api/ssh.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type Ssh struct {
```

Comment:
I think the general "Go style" is to use the all-caps version for things like HTTP or SSH. I was trying to find an article to link to, but my google-foo is failing me :smile: